.goreleaser.yaml

```
@@ -0,0 +1,103 @@
+builds:
+  # Linux
+  - id: tdexad-linux
+    main: ./cmd/tdexad
+    ldflags:
+      - -s -w
+    goos:
+      - linux
+    goarch:
+      - amd64
+    binary: tdexad-linux
+
+  # Darwin
+  - id: tdexad-darwin
+    main: ./cmd/tdexad
+    ldflags:
+      - -s -w
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    binary: tdexad-darwin
+
+  ## CLI Linux
+  - id: tdexa-linux
+    main: ./cmd/tdexa
+    ldflags:
+      - -s -w
+    goos:
+      - linux
+    goarch:
+      - amd64
+    binary: tdexa-linux
+
+  - id: tdexa-darwin
+    main: ./cmd/tdexa
+    ldflags:
+      - -s -w
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    binary: tdexa-darwin
+
```

Since we do not need to split for CGO, we can just make a single one for all goos and arch

```suggestion
  - id: tdexad
    main: ./cmd/tdexad
    ldflags:
      - -s -w
    goos:
      - linux
      - darwin
    goarch:
      - amd64
      - arm64
    binary: tdexad
    
  - id: tdexa
    main: ./cmd/tdexa
    ldflags:
      - -s -w
    goos:
      - linux
      - darwin
    goarch:
      - amd64
      - arm64
    binary: tdexa
```

---

Done